.eslintrc.json

```diff
@@ -45,6 +45,7 @@
     "no-empty": ["error", { "allowEmptyCatch": true }],
     "no-plusplus": "off",
     "no-restricted-syntax": "off",
+    "no-use-before-define": "off",
```

Comment:
👎

Reply:
👍 `no-use-before-define` is a TS error now when it matters. eslint can yield false positives.